service-front/assets/js/moj/moj.modules/moj.analytics.js

```diff
@@ -4,64 +4,39 @@
 (function () {
   'use strict';
 
-  if (typeof gaConfig === 'undefined') {
-    moj.log('gaConfig not set. skipping Google Analytics tracking.');
-    return;
-  }
-
   moj.Modules.Analytics = {
-    init: function () {
-      this.bindEvents();
+    dataLayer: window.dataLayer || [],
+    gaId: '1DVC295G9L',
```

Comment:
ah, I see, it probably does

Reply:
Yep always the same, but this ID is what I'll be changing once this PR passes code review, to the production one in GA